superbuild/projects_modules/medInria.cmake

```diff
@@ -92,20 +86,16 @@ set(cmake_cache_args
   -DRPI_ROOT:PATH=${RPI_ROOT}
   -DTTK_ROOT:PATH=${TTK_ROOT}
   -DVTK_ROOT:PATH=${VTK_ROOT}
-  -DQt${QT_VERSION_MAJOR}_ROOT:PATH=${Qt${QT_VERSION_MAJOR}_ROOT}  
-
-  
-  -DDCMTK_DIR:PATH=${DCMTK_DIR}
-  -Ddtk_DIR:PATH=${dtk_DIR}
```

Comment:
It's better to keep *_dir path variables to provide compatibility with external plugins.